### PR TITLE
database.sql file with initialisation - features a group set up with …

### PR DESCRIPTION
**Pull request by DanGRT**
Please review to ensure db initialisation is correct

### CODE REVIEW

File: database.sql [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  id serial,
+  member_id INT NOT NULL,
+  event_id INT NOT NULL,
+  PRIMARY KEY (id),
+  FOREIGN KEY (member_id) REFERENCES member (id),
+  FOREIGN KEY (event_id) REFERENCES event (id)
+
+);
+
+CREATE TABLE suggestions (
```
Comment (joelamb) [inline database.sql]: should be singular 'suggestion' for table name

Comment (DanGRT) [reply database.sql]: Will correct

File: database.sql [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  id SERIAL,
+  venue_name VARCHAR(50) NOT NULL,
+  reason VARCHAR(50) NOT NULL,
```
Comment (joelamb) [inline database.sql]: data type could be TEXT to allow users to leave longer comments

File: database.sql [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  suggester_id INT NOT NULL,
```
Comment (joelamb) [inline database.sql]: I would use member_id to make the connection more immediate

File: database.sql [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  event_id INT NOT NULL,
+  PRIMARY KEY (id),
+  FOREIGN KEY (member_id) REFERENCES member (id),
+  FOREIGN KEY (event_id) REFERENCES event (id)
+
+);
+
+CREATE TABLE suggestions (
+  id SERIAL,
+  venue_name VARCHAR(50) NOT NULL,
```
Comment (joelamb) [inline database.sql]: Neeed a  postcode field

**Pull request merged**